wger/core/migrations/0010_auto_20170330_1116.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.12 on 2017-03-30 08:16
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
```

Comment:
Yes it is, could you try putting the commands like `pip install pep8` using markdown, it makes them more readable and like distinct.